_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,271 @@
+---
+id: e88epe
+name: Image not in the accessibility tree is decorative
+rule_type: atomic
+description: |
+  This rule checks that visible img, svg and canvas elements that are ignored by assistive technologies are decorative
+accessibility_requirements:
+  wcag20:1.1.1: # Non-text content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - Accessibiliy tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+  assets:
+    - W3C
+    - Wikimedia
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+Any `img`, `canvas` or `svg` element that is [visible][] and is not [included in the accessibility tree][], except if one of the following is true:
+
+- **unloaded img**: the element is an `img` and it's [current request][]'s [state][image request state] is not [completely available][]; or
+- **ignored svg**: the element is an `svg` with an empty (`""`) [accessible name][] and a [semantic role][] of `graphics-document`; or
+- **ignored canvas**: the element is a `canvas` with an empty (`""`) [accessible name][] and no [explicit semantic role][]; or
```

Comment:
I understand the assumption, but I can't really understand why should these be inapplicable. If they are not in the accessibility tree, shouldn't they be descriptive?

Btw, I tested an `svg` with VoiceOver on Firefox and it was not ignored. It was ignored with VoiceOver on Chrome.

Reply:
I agree there is something funky with the applicability

I read it the same way as @carlosapaduarte, but @WilcoFiers's answer "[these] elements that are included in the accessibility tree but (…) ignored by AT" makes little sense with that interpretation: if these elements are included in the accessibility tree, then they are already inapplicable and don't need to be specifically excluded.

Reply:
Jup, rereading this, I see your point. Fixing it.